package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "@eppo/sdk-test-data",
+  "version": "1.0.0",
+  "repository": "git@github.com:Eppo-exp/sdk-test-data.git",
+  "author": "",
+  "license": "MIT",
+  "engines": {
+    "node": ">=18.x"
+  },
+  "scripts": {
+    "validate:tests": "ts-node ./scripts/validateTests"
```

Comment:
from my experience, `ts-node` can be incredibly slow. might not be a concern here, but i'd probably `tsc` it first and then run the js file as normal. not blocking though

Reply:
Yeah, it's slow with larger projects, since you need to compile the TypeScript on the fly. There isn't much TypeScript right now, but if it grows we can always update it.